src/pages/Home.js

```diff
@@ -19,11 +19,16 @@ function Home({ t }) {
       <div
         data-aos="fade-right"
         className="absolute  "
-        style={{ top: '175vw' }}
+        style={{ top: '176vw' }}
```

Comment:
@JanaZR9 please don't use absolute positioning, you can use negative margins. Also, fix the sign-in button. It has same problem

If the image is getting shifted, you can give it a negative margin on top to pull it up. Bu avoid using absolute positioning!